docs/faq.md

```diff
@@ -64,4 +64,12 @@ To prevent data loss, SlateDB's `put()` API will block until the data has been f
 
 ## Does SlateDB support column families?
 
-SlateDB does not support [column families](https://github.com/facebook/rocksdb/wiki/column-families). Opening multiple SlateDB databases is cheap. This is what we recommend if you need to separate data.
\ No newline at end of file
+SlateDB does not support [column families](https://github.com/facebook/rocksdb/wiki/column-families). Opening multiple SlateDB databases is cheap. This is what we recommend if you need to separate data.
+
+## Are there any limits to key and value sizes?
+
+Keys are limited to a maximum of 65 KiB (65,535 bytes). Values are limited to a maximum of 4 GiB (4,294,967,295 bytes). Larger values require more memory and will take longer to write, so we recommend testing performance at your expected value size to ensure it meets your requirements.
+
+## Why does SlateDB have a WAL if it batches writes?
```

Comment:
There is another question earlier that covers the same topic in line 17. Can these two questions be merged?

Reply:
Hah, good catch! I'll remove the second one. I totally forgot I already had it. 😅

Reply:
Fixed: 16b782d